src/components/Scroller.tsx

```diff
@@ -0,0 +1,24 @@
+import { HTMLProps } from 'react'
+
+interface ScrollerProps {
+  scrollbars?: 'vertical' | 'horizontal' | 'both'
+}
+
+export function Scroller({
+  scrollbars = 'both',
+  ...props
+}: ScrollerProps & HTMLProps<HTMLDivElement>) {
+  return (
+    <div
+      {...props}
+      style={{
+        ...props.style,
+        ...(scrollbars === 'vertical' && { overflowY: 'auto' }),
+        ...(scrollbars === 'horizontal' && { overflowX: 'auto' }),
+        ...(scrollbars === 'both' && { overflow: 'auto' }),
+        scrollbarWidth: 'thin',
+        scrollbarColor: 'var(--gray-a8) var(--gray-a3)',
```

Comment:
The first color is for the thumb, the second is for the track. The latter doesn't work on Mac with the standard settings